expression/aggregation/util_test.go

```diff
@@ -1,22 +1,18 @@
 package aggregation
 
 import (
+	"testing"
 	"time"
 
-	"github.com/pingcap/check"
 	"github.com/pingcap/tidb/sessionctx/stmtctx"
 	"github.com/pingcap/tidb/types"
+	"github.com/stretchr/testify/require"
 )
 
-var _ = check.Suite(&testUtilSuite{})
-
-type testUtilSuite struct {
-}
-
-func (s *testUtilSuite) TestDistinct(c *check.C) {
+func TestDistinct(t *testing.T) {
 	sc := &stmtctx.StatementContext{TimeZone: time.Local}
```

Comment:
I guess this case can be run in parallel. Please confirm.

```suggestion
func TestDistinct(t *testing.T) {
	t.Parallel()
	sc := &stmtctx.StatementContext{TimeZone: time.Local}
```

Reply:
Fixed it - ffaa1a9